corporate/lib/stripe.py

```diff
@@ -310,13 +310,12 @@ def next_invoice_date(plan: CustomerPlan) -> datetime | None:
     if plan.status == CustomerPlan.ENDED:
         return None
     assert plan.next_invoice_date is not None  # for mypy
-    months_per_period = 1
-    periods = 1
-    dt = plan.billing_cycle_anchor
-    while dt <= plan.next_invoice_date:
-        dt = add_months(plan.billing_cycle_anchor, months_per_period * periods)
-        periods += 1
-    return dt
+    months = 1
+    datetime = plan.billing_cycle_anchor
```

Comment:
`datetime` is a risky local variable name since it's the name of a Python module. I think we should just use something more semantic about which time this is, like `candidate_invoice_date`

Reply:
Updated for that suggested variable name.